ROADMAP.md

```diff
@@ -27,17 +27,17 @@ Note that Barcelona, which the mac-nosip connector uses, is no longer maintained
 | Media/files                      | ✔️  | ✔️        | ✔️          |
 | Replies                          | ✔️  | ✔️        | ✔️          |
 | Tapbacks                         | ✔️  | ✔️        | ✔️          |
-| Edits                            | ❌   | ❌         | ❌           |
-| Unsends                          | ❌   | ❌         | ❌           |
+| Edits                            | ❌   | ❌         | ✔️ (BlueBubbles Server >= 1.9.6)           |
+| Unsends                          | ❌   | ❌         | ✔️ (BlueBubbles Server >= 1.9.6)           |
 | Own read receipts                | ✔️  | ✔️        | ✔️          |
 | Other read receipts              | ✔️  | ✔️        | ✔️          |
 | Typing notifications             | 🛑  | ✔️        | ✔️          |
 | User metadata                    | ✔️  | ✔️        | ✔️          |
 | Group metadata                   | ✔️  | ✔️        | ✔️          |
 | Group Participants Added/Removed | ✔️  | ✔️        | ✔️          |
-| Backfilling history              | ✔️  | ✔️        | ✔️‡         |
+| Backfilling history              | ✔️‡  | ✔️‡        | ✔️‡         |
 
-‡The BlueBubbles connector doesn't support backfilling tapbacks yet
+‡ Backfilling tapbacks is not yet supported
```

Comment:
This change looks wrong

Reply:
Unless I am misreading something, your comment in the backfilling code would apply to all modes of this bridge, not just BlueBubbles. And my code is returning tapbacks during the backfill, so it is this code that isn't processing them.
https://github.com/mautrix/imessage/blob/6118c551ac9d840e6ca2e5ab1c30efc8f60549c3/historysync.go#L166